codelabs/intro.md

```diff
@@ -2,4 +2,55 @@
 sidebar_position: 3
 ---
 
-# Intro
\ No newline at end of file
+# Intro
+
+
+# Multipaz
+
+This repository contains libraries and applications for working with real-world
+identity. The initial focus for this work was mdoc/mDL according to [ISO/IEC 18013-5:2021](https://www.iso.org/standard/69084.html)
+and related standards but the current scope also include other credential formats and
+presentment protocols.
+
+## Multipaz Libraries
```

Comment:
To keep things concise and since this information is already covered in the project overview, consider adding a brief introduction here and linking to the section where the full content is provided.